doc/tutorials/content/sources/executor/executor.cpp

```diff
@@ -0,0 +1,53 @@
+#include <pcl/common/generate.h>
+#include <pcl/filters/functor_filter.h>
+#include <pcl/visualization/cloud_viewer.h>
+
+int
+main()
+{
+  // Fill the cloud with randomly generated points
+  auto cloud = pcl::make_shared<pcl::PointCloud<pcl::PointXYZ>>();
+  pcl::common::CloudGenerator<pcl::PointXYZ, pcl::common::UniformGenerator<float>>
+      generator{{-20., 20., 128}};
+  generator.fill(200, 200, *cloud);
+
+  // Create executors
+  auto simple_exec = executor::default_inline_executor();
+  auto parallel_exec = executor::default_omp_executor();
```

Comment:
Hmm i just realized that this `default_omp_executor` is lacking in forwarding arguments to the objects constructor :/

Reply:
Didn't understand what you are trying to say?

Reply:
I would like to have declared `auto parallel_exec` as const. Think about what would be required for that to happen.